Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: build test clean run-dev release-snapshot run-docker run docker-compose-up docker-compose-down lint
+.PHONY: build test clean run-dev release-snapshot run-docker run docker-compose-up docker-compose-down lint docker-test
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)